### PR TITLE
Documentation: mention developer group in macOS install instructions

### PR DESCRIPTION
**Pull request by aarzilli**
Some users seem to need to do this in order to use a debugger.

### CODE REVIEW

File: Documentation/installation/README.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ If you didn't enable Developer Mode using Xcode you will be asked to authorize t
 sudo /usr/sbin/DevToolsSecurity -enable
 ```
 
+You might also need to add your user to the developer group:
+
+```
+sudo dscl . append /Groups/_developer GroupMembership your_username
+```
```
Comment (icholy) [inline Documentation/installation/README.md:45]: Can `your_username` be replaced with `$(whoami)`?

Comment (derekparker) [reply Documentation/installation/README.md:45]: Second this, seems cleaner from a copy/paste perspective for users.

Comment (aarzilli) [reply Documentation/installation/README.md:45]: Done.

**Review by derekparker — APPROVED** [review]
LGTM

**Pull request merged**